public/scss/home.scss

```diff
@@ -0,0 +1,238 @@
+        transition: background-color 0.6s ease;
+      }
+
+      .active,
+      .dot:hover {
+        background-color: #717171;
+      }
+    }
+  }
+  section {
+    position: relative;
+    padding: 23px 0;
+    margin-bottom: 20px;
+    background-color: #fff;
+    &:nth-child(odd) {
+      .cat_img {
+        order: 1;
+      }
+    }
+    &:after {
+      content: "";
+      position: absolute;
+      left: 50%;
+      top: -33px;
+      width: 60%;
+      height: 5px;
+      background: #fff;
+      border-radius: 290px / 8px;
+      -moz-border-radius: 290px / 8px;
+      -webkit-border-radius: 290px / 8px;
+      box-shadow: 0 5px 15px rgba(0, 0, 0, 0.9);
+      -moz-box-shadow: 0 10px 17px rgba(0, 0, 0, 0.9);
+      -webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.9);
+      z-index: -3;
+      -webkit-transform: translate(-50%, -50%);
+      -ms-transform: translate(-50%, -50%);
+      transform: translate(-50%, -50%);
+    }
+  }
+  .flex_boxes {
+    position: relative;
+
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    background-color: #fff;
+    z-index: 0;
+    .cat_img {
+      flex: 0 1 33%;
+    }
+    .cat_desc {
+      flex: 0 1 67%;
+      text-align: center;
+      padding: 0 30px;
+      p {
+        padding-bottom: 30px;
+        line-height: 1.4;
+        font-size: 18px;
+      }
+    }
+  }
+}
+
+/* Next & previous buttons */
+
+/* Caption text */
+.text {
+  color: #f2f2f2;
+  font-size: 15px;
+  padding: 8px 12px;
+  position: absolute;
+  bottom: 8px;
+  width: 100%;
+  text-align: center;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #f2f2f2;
+  font-size: 12px;
+  padding: 8px 12px;
+  position: absolute;
+  top: 0;
+}
+
+/* The dots/bullets/indicators */
+
+/* Fading animation */
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+/* 88888888888888888    slider   8888888888888888888888 */
+
+h2 {
+  line-height: 1;
+  margin-bottom: 15px;
+}
+
+@media only screen and (max-width: 1023px) {
+  .home {
+    .flex_boxes {
+      .cat_desc {
+        p {
+          font-size: 14px;
+          padding-bottom: 18px;
+        }
+      }
+    }
+    .slider {
+      .slider_home {
+        .dots {
+          bottom: 10px;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .home {
+    .flex_boxes {
+      .cat_desc {
+        flex: 0 1 60%;
+        padding: 0;
+      }
+      .cat_img {
+        flex: 0 1 40%;
+      }
+    }
+  }
+  nav {
+    ul {
+      li {
+        &:last-child {
+          padding-right: 15px;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (max-width: 479px) {
```

Comment:
This is wrong, you should write/build applications in Mobile First way and not Desktop First. Please fix the same.

Reply:
Done